apps/bot/src/bot/handlers/trending.ts

```diff
@@ -0,0 +1,113 @@
+import { Context } from "telegraf";
+import { FastifyInstance } from "fastify";
+import { trendingService } from "../../services/trending.service";
+import { TRENDING_MESSAGES } from "../../constants/trending.constants";
+import { getTrendingKeyboard } from "../keyboards/trending-menu";
+
+interface BotContext extends Context {
+  userId?: string;
+}
+
+export async function trendingHandler(
+  ctx: BotContext,
+  _server: FastifyInstance
+) {
+  try {
+    const loading = await ctx.reply(TRENDING_MESSAGES.FETCHING);
+    const chatId = ctx.chat!.id;
+
+    await trendingService.loadApiPage(chatId, 0);
+
+    const st = trendingService.getState(chatId)!;
```

Comment:
```suggestion
    const state = trendingService.getState(chatId)!;
```
Tên biến không quá dài, nên viết rõ ra.